doc/lms.pgsql

```diff
@@ -1253,8 +1253,11 @@ CREATE TABLE vlans (
     description varchar(254) DEFAULT NULL,
     customerid smallint DEFAULT NULL
         CONSTRAINT vlans_customerid_fkey REFERENCES customers (id) ON DELETE SET NULL ON UPDATE CASCADE,
+    netnodeid smallint DEFAULT NULL
+        CONSTRAINT vlans_netnodeid_fkey REFERENCES netnodes (id) ON DELETE SET NULL ON UPDATE CASCADE,
     PRIMARY KEY (id),
-    CONSTRAINT vlans_ukey UNIQUE (vlanid, customerid)
+    CONSTRAINT vlans_customerid_ukey UNIQUE (vlanid, customerid),
+    CONSTRAINT vlans_netnodeid_ukey UNIQUE (vlanid, netnodeid)
```

Comment:
To dany VLAN może obsługiwać tylko jeden węzeł? W rzeczywistości mamy sytuacje, że węzeł potrafi mieć wiele vlanów ...

Reply:
To UNIQUE dla pary.
Chodzi o to żeby nie było dwóch wpisów - np. vlan: 100, nr węzła: 100.
No i zwróć uwagę że to jest vlanid a nie id wpisu.

Reply:
Masz rację - moja pomyłka.